webpack/module-minifier-plugin-5/.eslintrc.js

```diff
@@ -1,10 +0,0 @@
-// This is a workaround for https://github.com/eslint/eslint/issues/3458
```

Comment:
This file should have been renamed, right?

Reply:
Fixed, this was out of date with the upstream